Documentation/ceph-cluster-crd.md

```diff
@@ -313,12 +313,23 @@ The following are the settings for Storage Class Device Sets which can be config
 * `tuneDeviceClass`: If `true`, because the OSD can be on a slow device class, Rook will adapt to that by tuning the OSD process. This will make Ceph perform better under that slow device.
 * `volumeClaimTemplates`: A list of PVC templates to use for provisioning the underlying storage devices.
   * `resources.requests.storage`: The desired capacity for the underlying storage devices.
-  * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass. This StorageClass should provide a raw block device, multipath device, or logical volume. Other types are not supported.
+  * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass. This StorageClass should provide a raw block device, multipath device, or logical volume. Other types are not supported. If you want to use logical volume, plese see [known issue of OSD on LV-backed PVC](#known-issue-of-osd-on-lv-backed-pvc)
   * `volumeMode`: The volume mode to be set for the PVC. Which should be Block
   * `accessModes`: The access mode for the PVC to be bound by OSD.
 * `schedulerName`: Scheduler name for OSD pod placement. (Optional)
 * `encrypted`: whether to encrypt all the OSDs in a given storageClassDeviceSet
 
+#### Known Issue of OSD on LV backed PVC
+
+There is a critical flaw in OSD on LV-backed PVC. LVM metadata would corrupt if both host and OSD container may modify it simultaneously. For example, administorator (in host) might modify it, that backs PVC, while the OSD initialization process (in container) modify it too. Before fixing this problem, please keep the following in mind to reduce the probability of this problems as possible.
+
+- Don't create new OSD on LV-backed PVC
```

Comment:
I think the previous paragraph already implies not to use LVM, so you can remove this bullet.